modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApiController.mustache

```diff
@@ -0,0 +1,160 @@
+package {{package}};
+
+{{#imports}}import {{import}};
+{{/imports}}
+
+import io.swagger.annotations.*;
+import play.mvc.Controller;
+import play.mvc.Result;
+import java.util.List;
+import java.util.ArrayList;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.inject.Inject;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import swagger.SwaggerUtils;
+import javafx.util.Pair;
+
+{{#useBeanValidation}}
+import javax.validation.constraints.*;
+{{/useBeanValidation}}
+
+{{>generatedAnnotation}}
+@Api(value = "{{{baseName}}}", description = "the {{{baseName}}} API")
+{{#operations}}
+public class {{classname}}Controller extends Controller {
+
+    private {{classname}}ControllerImp imp;
+    private ObjectMapper mapper;
+
+    @Inject
+    private {{classname}}Controller({{classname}}ControllerImp imp) {
+        this.imp = imp;
+        mapper = new ObjectMapper();
+    }
+
+{{#operation}}
+
+    @ApiOperation(value = "{{{summary}}}", notes = "{{{notes}}}"{{#returnType}}, response = {{{returnType}}}.class{{/returnType}}{{#returnContainer}}, responseContainer = "{{{returnContainer}}}"{{/returnContainer}}{{#hasAuthMethods}}, authorizations = {
+    {{#authMethods}}@Authorization(value = "{{name}}"{{#isOAuth}}, scopes = {
+    {{#scopes}}@AuthorizationScope(scope = "{{scope}}", description = "{{description}}"){{#hasMore}},
+    {{/hasMore}}{{/scopes}}
+    }{{/isOAuth}}){{#hasMore}},
+    {{/hasMore}}{{/authMethods}}
+    }{{/hasAuthMethods}}, tags={ {{#vendorExtensions.x-tags}}"{{tag}}",{{/vendorExtensions.x-tags}} })
+    @ApiResponses(value = { {{#responses}}
+    @ApiResponse(code = {{{code}}}, message = "{{{message}}}"{{#returnType}}, response = {{{returnType}}}.class{{/returnType}}){{#hasMore}}, {{/hasMore}}{{/responses}} })
+    {{#hasParams}}
+    @ApiImplicitParams({
+        {{#allParams}}{{^isPathParam}}@ApiImplicitParam(name = "{{paramName}}", value = "{{{description}}}"{{#required}}, required = true{{/required}}{{#defaultValue}}, defaultValue = "{{{defaultValue}}}"{{/defaultValue}}, dataType = "{{{dataTypeForImplicitParam}}}", paramType = "{{>paramType}}"){{#hasMore}},
```

Comment:
Here this annotation declares all params that are not "path" because in play framewowrk, every parameters that are not "path" are not parameters, they are extracted from the request() object. For this reason, I need to use an implicitParams. You are right, this should be a baseName (everything in the annotation should be baseName I think!)